editions/tw5.com/tiddlers/wikitext/Formatting in WikiText.tid

```diff
@@ -1,19 +1,35 @@
 caption: Formatting
 created: 20131205155959399
-modified: 20220513120211686
+modified: 20230303210038241
 tags: WikiText
 title: Formatting in WikiText
 type: text/vnd.tiddlywiki
 
-Available character formatting includes:
+!! Overview
 
-* <code>&#96;code&#96;</code> for `code` (<<.icon $:/core/images/mono-line>>), alternatively <code>&#96;&#96;double backticks allow &#96;embedded&#96; single backticks&#96;&#96;</code><br><br>
-* `''bold''` for ''bold'' text (<<.icon $:/core/images/bold>>)<br><br>
-* `//italic//` for //italic// text (<<.icon $:/core/images/italic>>)<br><br>
-* `__underscore__` for __underscored__ text (<<.icon $:/core/images/underline>>)<br><br>
-* `^^superscript^^` for ^^superscripted^^ text (<<.icon $:/core/images/superscript>>)<br><br>
-* `,,subscript,,` for ,,subscripted,, text (<<.icon $:/core/images/subscript>>)<br><br>
-* `~~strikethrough~~` for ~~strikethrough~~ text (<<.icon $:/core/images/strikethrough>>)<br><br>
-* `@@highlight@@` for @@highlighted@@ text (no editor toolbar button), see [[Styles and Classes in WikiText]] for more formatting options
+Available character formatting in WikiText includes:
 
-See also: [[Code Blocks in WikiText]]
+|Wikitext | Button | Shortcut |Rendered Output |h
+|Double single quotes are used for `''bold text''`| <kbd><<.icon $:/core/images/bold>></kbd> | <kbd><<displayshortcuts "((bold))">></kbd> |Double single quotes are used for ''bold text'' |
+|Double slashes are used for `//italic text//`| <kbd><<.icon $:/core/images/italic>></kbd> | <kbd><<displayshortcuts "((italic))">></kbd> |Double slashes are used for //italic text//|
+|Double underlines are used for `__underlined text__`| <kbd><<.icon $:/core/images/underline>></kbd> | <kbd><<displayshortcuts "((underline))">></kbd>  |Double underlines are used for __underlined text__ |
```

Comment:
The *character* should be referred to as "underscore" and the text decoration feature should be referred to as "underline"